run_pacta_data_preparation.R

```diff
@@ -33,7 +33,6 @@ config <-
 
 asset_impact_data_path <- config$asset_impact_data_path
 factset_data_path <- config$factset_data_path
```

Comment:
```suggestion
factset_data_path <- config$factset_data_path
data_prep_outputs_path <- config$data_prep_outputs_path
```
required below
note: #160 may make this unnecessary in the future

Reply:
#160 is relevant in the future, I agree. Discarding this suggestion, sine `data_prep_outputs_path` is defined on further down in the script since #151 was merged in.

https://github.com/RMI-PACTA/workflow.data.preparation/blob/a88994706b0f7ffc7e99fdde58548c588744c24c/run_pacta_data_preparation.R#L73-L76